articles/DeployExistingDjangoProject.md

```diff
@@ -28,7 +31,7 @@ Here's an overview of the steps involved.
 
 1. Upload your code to PythonAnywhere
 2. Set up a virtualenv and install Django and any other requirements
-3. Set up your web app using manual config and your WSGI file.
+3. Set up your web app using **manual config** (choose **Django** if you want to create a new project) and your WSGI file.
```

Comment:
I would change this do "do not choose the *Django* option, that's for new projects only"